VERSION

```diff
@@ -0,0 +1 @@
+v0.3.2
```

Comment:
👌

Reply:
kinda of a pain in a way because now we have to keep this in mind when doing PRs and deployments... idk. We don't need this but I thought it might be a nice to have in the datadog view? Then again I'm not sure. 🤷

Reply:
yeah, let's give it a time to see if we use it for datadog then worth it 👍